velox/exec/Driver.cpp

```diff
@@ -38,7 +38,7 @@ DriverCtx::DriverCtx(
       task(_task) {}
 
 const core::QueryConfig& DriverCtx::queryConfig() const {
-  return task->queryCtx()->config();
+  return task->queryCtx()->queryConfig();
```

Comment:
I don't think we need to rename config to queryConfig? as it is a member of queryCotx? Thanks!

Reply:
> I don't think we need to rename config to queryConfig? as it is a member of queryCotx? Thanks!

@xiaoxmeng, thanks for taking time to review the change.

TLDR: The config and queryConfig are very different objects.

More verbose:
QueryCtx is Context, which is BaseConfigManager, which has the following member:
```
std::shared_ptr<const Config> config_;
```
And that member actually possesses all the config settings.

While QueryCtx had this member:
```
QueryConfig config_;
```
Where QueryConfig does not actually have anything, but a simple wrapper around BaseConfigManager with a bunch of helper methods to fetch settings.

It was all messed up and hard to follow and confused me for quite a long time until I finally found some time to understand what is what in that mess.

I've just changed the names to reflect what they are. :)

Reply:
@spershin thanks for the explanation!